src/main/resources/application.yaml

```
@@ -1,3 +1,16 @@
+    show-sql: true
+    hibernate:
+      ddl-auto: update
+  docker:
+    compose:
+      lifecycle-management: start_only
+
+
+jwt:
+#  passtokenpasstokenpasstokenpasstokenpasstokenpasstoken
+  secretKey: cGFzc3Rva2VucGFzc3Rva2VucGFzc3Rva2VucGFzc3Rva2VucGFzc3Rva2VucGFzc3Rva2Vu
+  expiration: 3000 # 3000?
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)